pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -62,6 +65,7 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	cmd.Flags().VarP(&config.ImageStreams, "image", "i", "Name of an image stream to to use as a builder.")
 	cmd.Flags().Var(&config.DockerImages, "docker-image", "Name of a Docker image to use as a builder.")
 	cmd.Flags().StringVar(&config.Name, "name", "", "Set name to use for generated build artifacts")
+	cmd.Flags().VarP(&config.Environment, "env", "e", "Specify key value pairs of environment variables to set into each builder container.")
```

Comment:
No those are envVars in the DeploymentConfig. 
Check https://github.com/openshift/origin/blob/master/pkg/build/api/types.go#L244